### ISSUE TITLE
MSID: 88117 Version: 2 DOI: 2021.12.28.21268440

### ISSUE THREAD

**Issue by nlisgo** [elifesciences/enhanced-preprints-import#1034]
[PLACE MANUSCRIPT AND EDITOR DETAILS HERE WHEN AVAILABLE] See step 3

[PLACE PDF URL HERE WHEN AVAILABLE] See step 7

## Step 1. Inform bioRxiv

Who can help: @QueenKraken, @nlisgo, @scottaubrey

- [x] Manuscript is in data hub index (https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v1/by-publisher/elife/get-by-manuscript-id?manuscript_id=88117)

## Step 2. Create preview of manuscript

Who can help: @fred-atherden, @nlisgo, @scottaubrey

- [ ] Manuscript is available for preview (https://prod--epp.elifesciences.org/preview/88117/v2)
    - [ ] Trigger import for staging: https://prod--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://prod-elife-epp-data/data
    - [ ] Trigger import for staging: https://staging--epp.elifesciences.org/import
    - [ ] Sync data folder with s3://staging-elife-epp-data/data
    - [ ] Pull request created to [enhanced-preprint-data](https://github.com/elifesciences/enhanced-preprints-data/pulls)
    - [ ] Post link to pull request below

Pull request: [PLACE LINK TO PULL REQUEST HERE]

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-manuscript

## Step 3: Awaiting public reviews and QC

Who can help: Production team

- [ ] QC passed or QC not needed

Who can help: Editorial team

- [x] Public reviews available on sciety (https://sciety.org/articles/activity/10.1101/2021.12.28.21268440)
- [ ] Manuscript and editor details at the top of this issue (Supplied by Editorial team)
- [x] Reviews returned by EPP - https://prod--epp.elifesciences.org/api/reviewed-preprints/88117/v2/reviews

<details>
<summary>Example</summary>

```
"msas": "Genetics and Genomics", "Neuroscience"
"msid": "84628"
"version": "1"
"preprintDoi": "10.1101/2022.10.28.514241"
"articleType": "Reviewed Preprint"
"status": "Published from the original preprint after peer review and assessment by eLife."

"Reviewed Preprint posted": "2023-01-02"
"Sent for peer review": "2022-10-28"
"Posted to bioRxiv": "2022-11-21" (link: "Go to bioRxiv": "https://www.biorxiv.org/content/10.1101/2022.10.28.514241v1")

Editors:
Reviewing Editor
Michael B Eisen
University of California, Berkeley, United States
Senior Editor
Michael B Eisen
University of California, Berkeley, United States
```

</details>

## Step 4: Modify manuscripts.json (no PDF)

- [ ] Reviewed preprint is published to EPP (https://elifesciences.org/reviewed-preprints/88117)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below
- [ ] Request that a doi be registered
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

Pull request: [PLACE LINK TO PULL REQUEST HERE]
[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

<details>
<summary>Instructions to modify manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/publish-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/334/files

Once the pull request is merged in it should be available a few minutes later.

</details>

### Request that a doi

Post the following in [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77):

```
@Fred can you register a doi for https://elifesciences.org/reviewed-preprints/88117
```

## Step 5: Awaiting search reindex

- [ ] Reviewed preprint is avaliable on journal homepage (https://elifesciences.org)

The search reindex is triggered once an hour. We need the reviewed preprint to be indexed as the search application serves the journal homepage.

<details>
<summary>Additional info</summary>

If needed, the jenkins pipeline to reindex search can be triggered sooner.

https://alfred.elifesciences.org/job/process/job/process-reindex-reviewed-preprints/

</details>

## Step 6: Published! Request PDF generation

- [ ] Post the link to the PDF url at the top of the issue
    - [ ] Post link to [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment below

[#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) comment thread: [PLACE LINK TO COMMENT HERE]

Post the following to the [#enhanced-preprint](https://elifesciences.slack.com/archives/C03EVJSUA77) on slack:

```
@Ryan Dix-Peek please can you generate a PDF for https://elifesciences.org/reviewed-preprints/88117
```

## Step 7: Introduce PDF to data folder and git repo

- [ ] PDF is avaliable at https://github.com/elifesciences/enhanced-preprints-data/raw/master/data/88117/v2/88117-v2.pdf

Detailed instructions: https://github.com/elifesciences/enhanced-preprints-data#add-a-pdf

## Step 8: Add PDF url to manuscripts.json

- [ ] Reviewed preprint PDF is available for download (https://prod--epp.elifesciences.org/reviewed-preprints/88117)
    - [ ] Pull request created to [enhanced-preprint-client](https://github.com/elifesciences/enhanced-preprints-client/pulls)
    - [ ] Pull request switched to "Ready for review"
    - [ ] Post link to pull request below

[PLACE LINK TO PULL REQUEST HERE]

<details>
<summary>Instructions to add PDF url to manuscripts.json</summary>

- Visit: https://github.com/elifesciences/enhanced-preprints-client/actions/workflows/add-pdf-url-to-manuscript.yaml
- Click: Run workflow
- Complete the form and click "Run workflow"
- A successful run should result in a new pull request at https://github.com/elifesciences/enhanced-preprints-client/pulls
- Open the pull request and click the "Ready for review" button to trigger tests
- Once the tests pass and you are happy with the changes the PR can be merged

Example pull request: https://github.com/elifesciences/enhanced-preprints-client/pull/397/files

Once the pull request is merged in it should be available a few minutes later.

</details>

## Step 9: Done!

- [ ] Kettle is on!

**Comment by KJT08** [comment]
@fred-atherden reviews have been posted

**Comment by fred-atherden** [comment]
Thanks @KJT08. Do you now why this one was moved to `Create preview of manuscript`? Looks like it was loaded 2 weeks ago. I'll move it to `Modify manuscripts.json`

**Comment by KJT08** [comment]
Sorry @fred-atherden my mistake, it was meant to go to modify manuscripts.json

**Comment by fred-atherden** [comment]
No worries - just checking I wasn't missing something!

**Comment by fred-atherden** [comment]
@KJT08, @acollings any idea what's going on with the reviews for this one? 

They're not in EPP (https://prod--epp.elifesciences.org/api/reviewed-preprints/88117/v2/reviews), because they're not in [the DocMap](https://data-hub-api.elifesciences.org/enhanced-preprints/docmaps/v2/by-publisher/elife/get-by-manuscript-id?manuscript_id=88117).

**Comment by fred-atherden** [comment]
Preprint is QC OK but blocked for the above issue.

**Comment by acollings** [comment]
Hi @fred-atherden the reviews are on bioRxiv and Sciety (https://sciety.org/articles/activity/10.1101/2021.12.28.21268440), but not in Docmaps yet. They were posted yesterday so I'd expect them to be showing up by now??

**Comment by fred-atherden** [comment]
OK thanks Andy. 

@HazalCiplak - please could you take a look when you get a chance? I'm not sure what's going on with these reviews (why they're not int he DocMap).

**Comment by HazalCiplak** [comment]
Hello,
I've identified the issue. It appears that the preprint server DOI doesn't align with the manuscript version R1, which is why we're not seeing the reviews in DocMaps (please find related screenshot below). To resolve this, I've added a match for the preprint server DOI with the manuscript ID in the [Google Spreadsheet](https://docs.google.com/spreadsheets/d/15QcK8w-ssB7109RQEDtFpJPZ0J5HTGxoHa_2TtpMBbg/edit#gid=1336081641) and rerun the pipeline. This update should bring DocMaps back in sync.

<img width="1314" alt="Screenshot 2023-09-06 at 04 34 49" src="https://github.com/elifesciences/enhanced-preprints-import/assets/6259479/f6ed584f-49b5-45eb-8fd1-2b1f304a96a4">

**Comment by fred-atherden** [comment]
Many thanks Hazal! You've unblocked the ticket for us - so this one is ready to publish 🎉


Do you know why a match wasn't made in this case? It looks like the authors added a URL to their preprint in eJP, rather than a DOI. Could that be the reason?


![Image](https://github.com/elifesciences/enhanced-preprints-import/assets/43879983/cc05bdd2-9ccd-4519-a728-42efa41a3864)



Or is it something more involved (like titles not exactly matching or similar)?

Hopefully the changes we'll make to eJP that we previously discussed (ensuring that the authors provide a valid DOI and a version number) will ensure there are fewer of these cases in the future.

**Comment by HazalCiplak** [comment]
> Many thanks Hazal! You've unblocked the ticket for us - so this one is ready to publish 🎉

Great news!

> Do you know why a match wasn't made in this case? It looks like the authors added a URL to their preprint in eJP, rather than a DOI. Could that be the reason?

Sure, I'll investigate this issue more thoroughly to pinpoint where it might be going wrong. It could be related to the eJP export or the views I've set up to match the data. I'll create a separate issue for this. Also, it's good to note that authors in eJP are now providing a valid DOI and a version number, which should help improve data accuracy.

**Comment by fred-atherden** [comment]
Great - many thanks Hazal. 

Sorry I didn't make this clear - eJP haven't yet done the work to change the submission form so that authors provide a valid DOI and version number, but it's in the pipeline!
